app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -85,6 +86,14 @@ angular.module("openshiftConsole")
           scope.persistentVolumeClaimForm.capacity.$setValidity('limitRangeMax', maxValid);
         };
 
+        var validateQuota = function() {
+          var requestedStorage = scope.claim.amount && usageValue(scope.claim.amount + scope.claim.unit);
+          var outOfClaims = QuotaService.isAnyStorageQuotaExceeded(scope.quotas, scope.clusterQuotas);
+          var outOfStorage = QuotaService.willRequestStorageQuotaBeExceeded(scope.quotas, scope.clusterQuotas, requestedStorage);
```

Comment:
so pass scope.claim.amount + scope.claim.unit here instead of the converted value, and let the quota service method handle the usageValue conversion stuff